scripts/submit_emod3d.py

```diff
@@ -27,91 +27,103 @@
 from shared_workflow.shared_template import write_sl_script
 
 
-def main(args: argparse.Namespace, logger: Logger = get_basic_logger()):
-    params = utils.load_sim_params(os.path.join(args.rel_dir, "sim_params.yaml"))
+def main(
+    auto: bool = False,
+    machine: str = host,
+    ncores: int = platform_config[const.PLATFORM_CONFIG.HF_DEFAULT_NCORES.name],
+    rel_dir: str = ".",
+    retries: int = 0,
+    write_directory: str = None,
+    logger: Logger = get_basic_logger(),
+):
+    rel_dir = Path(rel_dir).resolve()
 
-    submit_yes = True if args.auto else confirm("Also submit the job for you?")
+    try:
+        params = utils.load_sim_params(rel_dir / "sim_params.yaml")
+    except FileNotFoundError:
+        logger.error(f"Error: sim_params.yaml doesn't exist in {rel_dir}")
+        raise
 
-    logger.debug("params.srf_file {}".format(params.srf_file))
-    # Get the srf(rup) name without extensions
-    srf_name = os.path.splitext(os.path.basename(params.srf_file))[0]
+    params.sim_dir = Path(params.sim_dir).resolve()
```

Comment:
Same comment as above. Especially when it is saved to a local variable a few lines down.

Reply:
done